### ISSUE TITLE
[Bug]: wrong consumption stats since 0.5 version

### ISSUE THREAD

**Issue by man010** [cjaliaga/home-assistant-aquarea#91]
### The problem

Prior to 0.5 version, the consumption the « heating accumulated consumption » and « tank accumulated consumption » were very accurate. From 0.5 it seems to be no longer the case.

FYI when I look at the consumption of yesterday, based on these 2 sensors I have consumed 64kWh, while my energy provider reports 59kWh.

i provide 2 pictures (Panasonic SmartCloud + a chart on HA based on these 2 sensors) on my consumption on Jan 7th and 2 pictures of my consumption of Jan 11th.

![IMG_3195](https://github.com/cjaliaga/home-assistant-aquarea/assets/98453674/9feebab6-b301-4d7b-82e1-c97c45ec88ed)
![IMG_3197](https://github.com/cjaliaga/home-assistant-aquarea/assets/98453674/5eaa03d6-c8bc-4065-a3b5-72f48d453303)
![IMG_3196](https://github.com/cjaliaga/home-assistant-aquarea/assets/98453674/32befc21-c624-4b11-b9c9-719ed7ac5c81)
![IMG_3199](https://github.com/cjaliaga/home-assistant-aquarea/assets/98453674/b69722d4-bf78-4dfd-8f69-da9ba1226700)

### What version of Home Assistant Core are you running?

core-2024.1.2

### What type of installation are you running?

Home Assistant OS

### What version of the integration are you running?

6.1.71-haos

### Device description

1 zone
It supports cooling, but we never use it
It has DHW
Is it connected with 3 live wires (380V)

### Is the integration loading?

Yes

### Is the adapter able to connect to the Aquarea Smart Cloud with a stable internet connection?

Yes

### Are you able to perform the action you're trying to with the Aquarea Smart Cloud Site/App?

Yes

### Area of the issue

Sensor values issues

**Comment by cjaliaga** [comment]
Thanks for the report @man010. Initially I wouldn't think it's related to 0.5.0 as there has not been any changes to the consumption sensors on that version, neither to [aioaquarea](https://github.com/cjaliaga/aioaquarea) 0.5. Perhaps is just a coincidence that the issue surfaced after the update.

Has the behavior continued? I'm working on adding extra debug logs to the consumption sensors so we can better debug these issues. I'll update here when the version is released so you can enable debug logs and see if the issue reproduces again.

**Comment by compala** [comment]
Hi, I can only confirm, I am attaching the statistics for the last 7 days. The worst thing is that the ratio of the values from the control panel (which I trust the most) and the values from HA is completely different every day...
![image](https://github.com/cjaliaga/home-assistant-aquarea/assets/80894151/02fd8676-5727-45df-b8ed-45b3b2ab2b4a)

**Comment by cjaliaga** [comment]
Thanks @compala. With control panel do you mean the physical panel? 

Could you please let me know if the timezone on the cloud configuration is the same as Home Assistant? And if the device's hour is the same as in Home Assistant?

**Comment by compala** [comment]
Yup, i mean the physical panel. And yes, TZ and time are synchronized.
